src/components/NavBar/NavBar.css

```diff
@@ -1,6 +1,8 @@
 .containerNavBar {
```

Comment:
- [ ] Also, it is important we match the given design. Here is a screenshots of our app and the recommended design so that we can compare and fix it
Our App
![Screenshot from 2022-09-22 17-56-48](https://user-images.githubusercontent.com/46388113/191819601-1b06aa5f-e283-423a-8f89-3dab487b4300.png)

Design Recommended
![Screenshot from 2022-09-22 17-56-20](https://user-images.githubusercontent.com/46388113/191819548-6088ec6e-6459-47a5-ab37-374050239c5c.png)